Login.jsx

```diff
@@ -0,0 +1,31 @@
+import React, { useState } from 'react';
+import axios from 'axios';
+
+function Login() {
+  const [email, setEmail] = useState("");
+  const [password, setPassword] = useState("");
+
+  const handleSubmit = async (e) => {
+    e.preventDefault();
+    try {
+      const response = await axios.post('/api/login', { email, password });
+      if (response.data.success) {
+        window.location.href = "/home";
+      } else {
+        alert(response.data.message);
```

Comment:
Avoid using window.location to redirect users. This can lead to problems with how the React app maintains its internal state. Consider using React Router's useHistory hook instead.